### DISCUSSION TITLE
Custom directives on query not showing in schema definition

### DISCUSSION THREAD

**Discussion by Sandip888831** [Netflix/dgs-framework#1880] [General]
I have created custom authorization directive & annotated on the query:
```
getPatientDetails(id: String!): [User!]! @authorization(
        roles: ["REGISTERED"]
)
```

When I am calling the query, it's getting executed but I don't see it in schema definition.
I checked in /graphiql docs & also in introspection query _getFederationInfo { _service { sdl } }_, but I don't see it on query.
Please can someone help here.

**Comment by srinivasankavitha** [comment]
Have you declared it with `@DgsDirective`? https://netflix.github.io/dgs/advanced/custom-directives/
